src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1544,6 +1544,11 @@ elementFormDefault="qualified">
     <xs:element name="AnalysisLevel" type="msb:StringPropertyType" substitutionGroup="msb:Property"/>
     <xs:element name="InstallUrl" type="msb:StringPropertyType" substitutionGroup="msb:Property"/>
     <xs:element name="IsCodeSharingProject" type="msb:boolean" substitutionGroup="msb:Property"/>
+    <xs:element name="IsPackable" type="msb:boolean" substitutionGroup="msb:Property"/>
```

Comment:
@nkolev92 this look good? Should we add a description for it?

Reply:
I think it's pretty straightforward, but a description might be nice. 

I don't see the descriptions for the other bools, so not sure how verbose would a describe have to be. 
Adding @JonDouglas who's focusing on NuGet client experiences for help here. :)

Reply:
It's used for hovertext/autocomplete, so on the order of 1 sentence, like

https://github.com/dotnet/msbuild/blob/907fd2d02053a21d6044a4a8da4d85d6829e8652/src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd#L1905

https://github.com/dotnet/msbuild/blob/907fd2d02053a21d6044a4a8da4d85d6829e8652/src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd#L1735

Reply:
How about
`Indicates whether the project can be used to create a NuGet package.`

cc @JonDouglas

I'm ok if we don't do it/do it a future PR.

Reply:
I think this is fine. Other thoughts are:

`Indicates whether the project can be packed into a NuGet package. The default value is true.`

Reply:
I think the default depends on the project type. So it won't always be true. 

In particular I think test projects are marked `<IsPackable>false</IsPackable`. 

Sans that, I like your version of the description.

Reply:
We should update some of our docs then! I know there are edge cases like web projects, but is it generally true?

Reply:
Yeah, I think it's generally true because of the types of templates people use. 
classlib is probably most common and those are packable. 

We should definitely update our docs!